File: python/notes/loops.py
import random 
"""start = 0

while start <= 10:
    print (start)
    start += 2

import random

count = 1
goose = random.randint(1,10)

while count != goose: 
    print("DucK")
    count += 1 

print("GOOSE!")"""

number = random.randint(1,12038)

while True: 
    guess = int(input("Guess a number between 1-25:"))
    if guess == number:
        print(f"Yes, the number was {number}. You've won!")
        break #breaks takes us out of the loop and only works in loops
    elif guess > 12038:
        print("You've failed, that's not a option idiot. 1 through 25")
    elif guess < number:
        print("Gues higher")
    else:
        print("Guess lower")

File: C/notes/loops.c
//  while loops notes
#include <stdio.h>
// includes to get a random number
#include <stdlib.h>
#include <time.h>

int main(){
    int i = 1;
    while(i <=10){
        printf("%d\n", i);
        i++; // ++ increases by one
    }

    // generate  a random numb
    srand(time(NULL));

    printf("%d\n", (rand() %100) + 50);
    printf("%d\n", (rand() %4) + 1);
    printf("%d\n", (rand() %160) + 100);

    // example 2
    int goose = (rand() %9 + 1);
    int count = 1;

    while (count < goose ){
        printf("Duck!\n");
        count ++;
        }
    printf("Goose!\n");

    // example 3
    int timer = 30;
    while (timer > 0){
        printf("%d\n", timer);
        timer = timer -2;
    }
    printf("Times up");
}